frontend/src/pages/organization/AuditLogsPage/components/LogsFilter.tsx

```diff
@@ -1,28 +1,15 @@
 /* eslint-disable no-nested-ternary */
```

Comment:
when I click "Clear filters", it still shows the previous count of selected filters and I can't click on Apply anymore

<img width="637" alt="image" src="https://github.com/user-attachments/assets/adaa1da1-8a48-46fd-a338-08432157aae6" />